### PR TITLE
feat: userid, username in SQL Views [DHIS2-7742]

### PR DESCRIPTION
**Pull request by jimgrace**

### CODE REVIEW

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/sqlview/SqlView.java [hunk 1]
```diff
@@ -64,6 +64,12 @@
         "delete", "alter", "update", "create", "drop", "commit", "createdb",
         "createuser", "insert", "rename", "restore", "write" ).build();
 
+    public static final String CURRENT_USER_ID_VARIABLE = "_current_user_id";
+    public static final String CURRENT_USERNAME_VARIABLE = "_current_username";
+
+    public static final Set<String> BUILT_IN_VARIABLES = ImmutableSet.<String>builder().add(
```
Comment (larshelge) [inline dhis-2/dhis-api/src/main/java/org/hisp/dhis/sqlview/SqlView.java]: Shorten and use  `ImmutableSet.of`

Comment (larshelge) [inline dhis-2/dhis-api/src/main/java/org/hisp/dhis/sqlview/SqlView.java]: Rename to "standard variables" / `STANDARD_VARIABLES` ?

Comment (jimgrace) [reply dhis-2/dhis-api/src/main/java/org/hisp/dhis/sqlview/SqlView.java]: Changed.

Comment (jimgrace) [reply dhis-2/dhis-api/src/main/java/org/hisp/dhis/sqlview/SqlView.java]: Changed.

**Pull request merged**